includes/admin/class-wc-stripe-admin-notices.php

```
@@ -51,15 +68,17 @@ public function __construct() {
 	 * @param string $message     The notice message.
 	 * @param bool   $dismissible Whether the notice is dismissible.
 	 * @param array  $actions     Optional action buttons.
+	 * @param array  $css_rules   Optional CSS rules.
 	 *
 	 * @return void
 	 */
-	public function add_admin_notice( $slug, $class, $message, $dismissible = false, $actions = [] ) {
+	public function add_admin_notice( $slug, $class, $message, $dismissible = false, $actions = [], array $css_rules = [] ) {
```

PHPMD (2.15.0)

[error] The method add_admin_notice has a boolean flag argument $dismissible, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)